packages/monaco/src/browser/style/index.css

```diff
@@ -4,6 +4,13 @@
   font-size: inherit !important;
 }
 
+#quick-input-container {
+  position: absolute;
+  top: 0px;
+  right: 50%;
+  z-index: 1000000;
+}
+
```

Comment:
I'm wondering whether we need this change. It doesn't seem to do anything, and removing it doesn't change the behavior.